crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -513,6 +513,14 @@ where
             }
         }
     }
+
+    pub async fn periodic_check(&self) -> Result<()> {
```

Comment:
it would be helpful to keep the name to see what we're actually periodically checking

```suggestion
    pub async fn sub_channel_periodic_check(&self) -> Result<()> {
```